contrib/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
```

Comment:
I used `sh` instead of `bash` so we could add OSX testing in the future without too much trouble(you probably remember https://github.com/bitcoin-core/secp256k1/pull/750)